src/odemis/gui/win/acquisition.py

```diff
@@ -1187,23 +1191,30 @@ def _get_zstack_levels(self):
             zmax = focus_range[1]
         if zmax > focus_range[1]:
             # Too high => shift down
-            zmax -= zmax - focus_range[1]
-            zmin -= zmax - focus_range[1]
+            shift = zmax - focus_range[1]
+            zmin -= shift
+            zmax -= shift
+
         if zmin < focus_range[0]:
             # Too low => shift up
-            zmin += focus_range[0] - zmin
-            zmax += focus_range[0] - zmin
+            shift = focus_range[0] - zmin
+            zmin += shift
+            zmax += shift
 
         # Create focus zlevels from the given zsteps number
         zlevels = numpy.linspace(zmin, zmax, zsteps).tolist()
-        return zlevels
+        return zlevels, focus_value
+
+    def _fit_view_to_area(self):
```

Comment:
How does this work? line 1232 calls this method with an argument. There is no `.area` on `self`.
I think maybe it's a incorrect revert from my own changes to this class?

Reply:
Copy-paste the function as it is in the master. It will "magically" disappear from the PR.